components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -89,7 +89,9 @@ spec:
             - name: TYK_MDCB_SERVEROPTIONS_CERTIFICATE_KEYFILE
               value: "{{ .Values.mdcb.tls.certificateKeyFile }}"
             - name: TYK_MDCB_SERVEROPTIONS_CERTIFICATE_CERTFILE
-              value: "{{ .Values.mdcb.tls.certificateCertFile }}"                
+              value: "{{ .Values.mdcb.tls.certificateCertFile }}"
+            - name: TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET
```

Comment:
We now have two `TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET` environment variables declared. Assume that i have following `values.yaml` file:
```yaml
global:
  mdcbSynchroniser:
    enabled: true
  mtls:
    privateCertificateEncodingSecret: something
```
Now, the final MDCB deployment will have two `TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET` environment variable declaration.
We do not need both of them as they are same.